gcloud/dns/client.py

```diff
@@ -108,3 +108,29 @@ def zone(self, name, dns_name):
         :returns: a new ``ManagedZone`` instance
         """
         return ManagedZone(name, dns_name, client=self)
+
+    def get_zone(self, zone_name):
+        """Get a zone by name.
+
+        If the zone isn't found, this will raise a
+        :class:`gcloud.exceptions.NotFound`.
+
+        For example::
+
+          >>> try:
+          >>>   zone = client.get_zone('my-zone')
```

Comment:
We have `get_bucket` and `lookup_bucket`. The exception actually provides information.